code-server GA doc updates (#562)
opendatahub-io/opendatahub-documentation: modules/pushing-project-changes-in-code-server-to-a-git-repository.adoc

:_module-type: PROCEDURE

[id='pushing-project-changes-in-code-server-to-a-git-repository_{context}']
= Pushing project changes in code-server to a Git repository

[role='_abstract']
To build and deploy your application in a production environment, upload your work to a remote Git repository.

.Prerequisites

> * You have a launched and running code-server workbench.

Reviewer: You have a running code-server workbench.

> * You have opened a workbench in the code-server interface.

Reviewer 2: ```suggestion
* You have opened a workbench created with a code-server image.
```

Reviewer 2: maybe created _from_?

> * You have already added the relevant Git repository in code-server.

Reviewer: remove "already"

* You have permission to push changes to the relevant Git repository.
* You have installed the Git version control extension.

.Procedure

> . In your code-server workbench, from the Activity Bar, select the menu icon (image:images/codeserver-menu-icon.png[The Menu icon]) > *File* > *Save All*  to save any unsaved changes.

Reviewer: workbench? same comment as above
